src/re_frame/registrar.cljc

```diff
@@ -26,15 +26,15 @@
 
   ([kind id required?]
    (let [handler (get-handler kind id)]
-     (when debug-enabled?
-       (when (and required? (nil? handler))
+     (when debug-enabled?                                   ;; This is in a separate when so Closure DCE can run
```

Comment:
DCE is an abbreviation for *Dead Code Elimination*, a feature of the Google Closure compiler.